nengo/tests/test_cache.py

```diff
@@ -299,6 +299,13 @@ def test_unsupported_fingerprinting(obj):
         Fingerprint(obj)
 
 
+@pytest.mark.parametrize("obj", [
+    nengo.SpikingRectifiedLinear(),  # issue 1549
+])
+def test_supported_fingerprinting(obj):
+    assert Fingerprint.supports(obj)
```

Comment:
Not sure, how much value this tests add, but I guess there might be some as `supports` is doing a little bit more than testing list inclusion.

But shouldn't this test then be done for all whitelisted types?

Reply:
This was to satisfy the required guideline _"I have added tests to cover my changes."_ I can add some extra testing, but I was expecting someone might first have an idea to ensure this stays in sync with new models/etc.

Reply:
One option could be to create a whitelist decorator and then prepend that to anything we want in the whitelist. That way it is localized alongside the object being whitelisted, rather than hidden within `cache.py`.

Reply:
We could have this test test all `NeuronTypes` in `neurons.py`. Then if there's one that isn't in the whitelist, we should catch it right away.

Reply:
> This was to satisfy the required guideline _"I have added tests to cover my changes."_

I would think of this as a guideline, not a strict rule. Tests should add some actual benefit and not just satisfy this guideline.



> One option could be to create a whitelist decorator and then prepend that to anything we want in the whitelist.

That would make `neurons.py` depend on `cache.py` which might be a circular reference that I would like to try to avoid.